source/standards/supporting-services.html

```diff
@@ -0,0 +1,16 @@
+---
+title: Supporting services
+expires: 2017-10-01
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+The vision for support operations at GDS is to build a high quality, robust, resilient and consistent approach that is an exemplar for running digital services at scale.
+
+One important element is the development of the GDS incident management framework which focuses on restoring normal operations as quickly as possible with the least possible impact on users, and in the most cost-effective way.
+
+Our processes are designed to minimise recovery time for our services and enable iterative improvements by regular post-incident reviews. Each programme within GDS will define specific priority levels associated with the potential failure points for applications underlying their services.
```

Comment:
I think we should phrase things as second person rather than first person plural. Eg

> You should design your process to minimise recovery time for your service, and you should iterate your process through regular post-incident reviews.